content/pt-br/cluster.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Cluster
+status: Completed
+category: conceito
+---
+
+## O que é  
+Um cluster é um grupo de máquinas ou aplicações que trabalham juntos para um objetivo comum. No contexto da computação nativa em nuvem, o termo é mais frequentemente aplicado ao Kubernetes. Um cluster Kubernetes é um conjunto de serviços (ou cargas de trabalho) executados em seus próprios contêineres, geralmente em máquinas diferentes. O conjunto de todos esses serviços [contêinerizados](/containerization/), conectados em uma rede, representam um cluster.
+
+## Problema relacionado
+Um software executado em uma única máquina, apresenta um único ponto de falha, por exemplo, se essa máquina travar ou alguém desconectar acidentalmente o cabo de alimentação, algum sistema crítico para os negócios pode ser ficar offline. É por isso que os softwares modernos geralmente são construídos como [sistemas distribuídos](/distributed_apps/), agrupados em clusters.
```

Comment:
```suggestion
Um software executado em uma única máquina, apresenta um único ponto de falha, por exemplo, se essa máquina travar ou alguém desconectar acidentalmente o cabo de alimentação, algum sistema crítico para os negócios pode ficar offline. É por isso que os softwares modernos geralmente são construídos como [sistemas distribuídos](/distributed_apps/), agrupados em clusters.
```

We don't have `/distributed_apps/` path yet - I wonder if we should link to the en page for now? With `/containerization/` I think is fine since we have https://github.com/cncf/glossary/pull/487
Or `Distributed Apps` is also on the list of first terms to be translated?